packages/shared/lib/contexts/onboarding/helpers/restoreBackupForProfileManager.ts

```diff
@@ -20,6 +20,11 @@ export async function restoreBackupForShimmerClaimingProfileManagerHelper(
         await get(manager)?.setClientOptions(clientOptions)
     } catch (err) {
         console.error(err)
-        throw new UnableToRestoreBackupForProfileManagerError()
+        // Check if thrown err is because of an invalid password
+        if (err?.error.match(/`invalid stronghold password`/)) {
```

Comment:
We should put this into a `const` object for Stronghold-based errors or something, so we can use it elsewhere

Reply:
Already exists a ClientError object. Went ahead an added the `invalid stronghold password` error to it